### PR TITLE
Superhero Overhaul Part 2: Magnetic Boogaloo

### PR DESCRIPTION
**Pull request by ZomgPonies**
- Griffin
- [x] Give mask ability to use CAW sound and action button it
- [x] Make griffin start off with freedom implant

- Owlman 
- [x] Give mask ability to use HOOT sound and action button it
- [ ] Potentially give him ability to recruit a single sidekick (Hooter Girl)?
- [x] Make his belt respawn bolas/smokebombs slowly

- E-N
- [x] Add Electro-Negmatic (E-N, the magnet-based robotic supervillain).
- [x] Add Magnetic Disarm Power Thing. Spell?
- [x] Add permanent minor magnitis effect to the E-N costume

- Bugfixes
- [x] Fixed recruit sidekick showing up in shadowling panel
- [x] Make sure all superheros spawn with a basic radio
- [x] Make sure henchmen spawn with ID cards
- [x] Fix henchmen so they are black eyes, no beard, and caucasian skin color

### CODE REVIEW

**Comment by ZomgPonies** [comment]
Should be good for review. I'll add a sidekick to Owlman some other time when I have proper sprites.

File: code/datums/spells/magnet.dm [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/obj/effect/proc_holder/spell/wizard/targeted/magnet
+	name = "Magnetic Pull"
+	desc = "Pulls metalic objects from enemies hands with the power of MAGNETS."
+	charge_type = "recharge"
+	charge_max	= 300
+	clothes_req = 0
+	invocation = "UN'LTD P'WAH!"
```
Comment (TheDZD) [inline code/datums/spells/magnet.dm]: Might be worth giving it a different invocation than the lightning bolt.

**Comment by Krausus** [comment]
I'm not sure how I feel about a magnetic superhero who can't tear up nearby floor tiles and walls to use as projectiles.

File: code/datums/spells/magnet.dm (hunk 1 above)
Comment (ZomgPonies) [inline code/datums/spells/magnet.dm]: Doesn't really matter since it's not actually used. Invocation type is none.

**Pull request merged**